### PR TITLE
Fix-EZP-28398: Using Assetic versioning breaks Platform Admin UI

### PR DESCRIPTION
**Pull request by kmadejski**
JIRA issue: https://jira.ez.no/browse/EZP-28398

Fix for 1.7.x.

### CODE REVIEW

File: Resources/views/PlatformUI/shell.html.twig [hunk 1]
```diff
@@ -59,7 +59,8 @@
     <script>
         {{ ez_platformui_yui_config( "YUI.GlobalConfig" ) }}
 
-        var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "{{ asset("bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/") }}";
+        var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "/bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/";
+        {#var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "{{ asset("bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/") }}";#}
```
Comment (sunpietro) [inline Resources/views/PlatformUI/shell.html.twig]: Commented code

File: Resources/views/PlatformUI/shell.html.twig [hunk 2]
```diff
@@ -59,7 +59,7 @@
     <script>
         {{ ez_platformui_yui_config( "YUI.GlobalConfig" ) }}
 
-        var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "{{ asset("bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/") }}";
+        var CKEDITOR_BASEPATH = ALLOYEDITOR_BASEPATH = "/bundles/ezplatformuiassets/vendors/alloy-editor/dist/alloy-editor/";
```
Comment (andrerom) [inline Resources/views/PlatformUI/shell.html.twig]: is there some code anywhere prefixing this url with possible base path and so on or is it not needed here?

Comment (kmadejski) [reply Resources/views/PlatformUI/shell.html.twig]: No, there is no code which is prefixing this patch. It is used only by AlloyEditor when it loads dependencies (like `config.js`,  `styles.js` and so on).

**Comment by kmadejski** [comment]
@adamwojs your suggestion from https://github.com/ezsystems/PlatformUIBundle/pull/928#pullrequestreview-83496793 added also here.

**Comment by lserwatka** [comment]
@kmadejski please merge this up to 1.12, master then master to 2.0

**Pull request merged**